File: sams-backend/src/main/java/com/sams/entity/Attribute4.java
package com.sams.entity;

import jakarta.persistence.*;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.AllArgsConstructor;
import java.time.LocalDateTime;

@Entity
@Table(name = "m_attribute4", schema = "base")
@Data
@NoArgsConstructor
@AllArgsConstructor
public class Attribute4 {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "id")
    private Long id;

    @Column(name = "sr_attribute_id4")
    private Long srAttributeId4;

    @Column(name = "org_id")
    private Long orgId;

    @Column(name = "attribute4_name")
    private String attribute4Name;

    @Column(name = "active")
    private Boolean active;

    @Column(name = "created_by")
    private String createdBy;

    @Column(name = "created_dt_disp")
    private String createdDtDisp;

    @Column(name = "updated_by")
    private String updatedBy;

    @Column(name = "updated_dt_disp")
    private String updatedDtDisp;

    @Column(name = "org_name")
    private String orgName;

    @PrePersist
    protected void onCreate() {
       // Auto-generated
    }

    @PreUpdate
    protected void onUpdate() {
       // Auto-generated
    }
}

File: SAMSUI/src/app/Model/master/attribute4.ts
import { BaseModel } from "../base/baseModel";

export class Attribute4Model extends BaseModel {

    public srAttributeId4 ?: number;

    public orgId ?: number;

    public attribute4Name ?: string;

    public active ?: boolean;

    public createdBy ?:string;

    public createdDtDisp ?: string;

    public updatedBy ?: string;

    public updatedDtDisp ?: string;

    public pageNumber ?: number;

    public recordsPerPage ?: number;

    public orgName ?: string;

    public isListEmpty?:boolean;
}